tests/integration/components/bs-accordion-test.js

```diff
@@ -328,3 +328,103 @@ testBS4('yields change action to add custom behaviour', function(assert) {
     done();
   }, 500);
 });
+
+testBS3('clicking collapsed item with contextual title expands it', function(assert) {
+  this.render(hbs`{{#bs-accordion as |acc|}}
+    {{#acc.item value=1 as |aitem|}}
+      {{#aitem.title}}TITLE1{{/aitem.title}}
+      {{#aitem.body}}CONTENT1{{/aitem.body}}
+    {{/acc.item}}
+    {{#acc.item value=2}}CONTENT2{{/acc.item}}
+  {{/bs-accordion}}`);
+  let item = this.$('.panel:eq(0)');
+  let done = assert.async();
+
+  item.find('.panel-heading').click();
```

Comment:
We could unify the separate tests for BS3/BS4 here, by using a helper like this to get theBS version depending class name: https://github.com/kaliber5/ember-bootstrap/blob/master/tests/helpers/bootstrap-test.js#L62-L64. It seems the logic is identical, the difference are just class names. So by not hardcoding class names instead using some helper, we can have a single test implementations for both B3 and BS4.